src/events/ClientReady.ts

```diff
@@ -56,25 +55,24 @@ export default class ClientReadyEvent extends BaseEvent {
 			await logToChannel(mainGuild, process.env.ERROR_LOGS_CHANNEL_ID, {
 				embeds: [readyEmbed],
 			});
-		}
```

Comment:
This will cause Practice Sessions to stop working and is unrelated to the PR

Reply:
This should be fixed in https://github.com/Sachin-dot-py/r-igcse-bot/pull/129/commits/63437a638a74a1f527e421da9532f9ddf306d29a